services/bls_aggregation/blsagg.go

```diff
@@ -229,13 +229,13 @@ func (a *BlsAggregatorService) singleTaskAggregatorGoroutineFunc(
 	signedTaskRespsC <-chan types.SignedTaskResponseDigest,
 ) {
 	defer a.closeTaskGoroutine(taskIndex)
-
 	quorumThresholdPercentagesMap := make(map[types.QuorumNum]types.QuorumThresholdPercentage)
 	for i, quorumNumber := range quorumNumbers {
 		quorumThresholdPercentagesMap[quorumNumber] = quorumThresholdPercentages[i]
 	}
 	operatorsAvsStateDict, err := a.avsRegistryService.GetOperatorsAvsStateAtBlock(context.Background(), quorumNumbers, taskCreatedBlock)
 	if err != nil {
+		a.logger.Error("Task goroutine failed to get operators state from avs registry", "taskIndex", taskIndex, "err", err)
```

Comment:
@samlaf I added the error logs here for now since this is useful to add for in case anything fails